module-2/backend/week_one/Ash.md

```diff
@@ -0,0 +1,104 @@
+## Week One - Module 2 Recap
+
+List the five common HTTP verbs and what the purpose is of each verb.
+  Get= Read part of CRUD
+  Post= Create part of CRUD
+  Put= Update part of CRUD
+  Delete= Delete part of CRUD

```

Comment:
One more would be PATCH to get to the fifth verb. Generally used interchangeably with PUT.